File: server/java/src/main/java/com/onlinepayments/example/api/models/additionalpaymentactions/AdditionalPaymentActionResponse.java
package com.onlinepayments.example.api.models.additionalpaymentactions;

import com.onlinepayments.example.service.domain.common.enums.Status;
import com.onlinepayments.example.service.domain.common.StatusOutput;

public record AdditionalPaymentActionResponse(
        String id,
        Status status,
        StatusOutput statusOutput
) { }

File: server/dotnet/sdk-dotnet-example/Presentation/Models/AdditionalPaymentActions/AdditionalPaymentActionResponse.cs
using Business.Domain.AdditionalPaymentActions;
using Business.Domain.Enums.AdditionalPaymentActions;

namespace Presentation.Models.AdditionalPaymentActions;

public class AdditionalPaymentActionResponse
{
    public string? Id { get; set; }

    public Status? Status { get; set; }

    public StatusOutput? StatusOutput { get; set; }
}